content/en/blog/2023/outreachy-2023-01.md

```diff
@@ -0,0 +1,45 @@
+---
+title: Call for participation - Outreachy May 2023
+linkTitle: End User Discussions Jan 2023
```

Comment:
```suggestion
linkTitle: Outreachy CFP '23
```

Reply:
Accepted, I just spelled out CFP as 'call for participation', as CFP can also be read as 'call for papers'.